src/demo-app/autocomplete/autocomplete-demo.ts

```diff
@@ -3,6 +3,16 @@ import {FormControl, NgModel} from '@angular/forms';
 import 'rxjs/add/operator/startWith';
 import 'rxjs/add/operator/map';
 
+interface State {
+  code: string;
+  name: string;
+}
+
+interface StateGroup {
```

Comment:
Also can you fix the lint failure? https://travis-ci.org/angular/material2/jobs/266064783